prisondersdilemma.py

```diff
@@ -11,182 +11,260 @@
 from numpy.random import shuffle
 import numpy as np
 import random
+from win32api import GetSystemMetrics
```

Comment:
I read you are set on making an interactive GUI. I would advise you not to try and build this completely from scratch, but to find a supported library.